bolt/prices/v1/prices.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package bolt.prices.v1;
+
+import "bolt/assets/v1/assets.proto";
+import "google/protobuf/timestamp.proto";
+
+// PricesService: Prices venue service....
+service PricesService {
+  // Endpoint to fetch the price of the pair.
+  rpc GetPrice(GetPriceRequest) returns (GetPriceResponse);
+
+  // Endpoint to fetch multiple pairs prices in batch.
+  rpc GetPrices(GetPricesRequest) returns (GetPricesResponse);
+
+  // Endpoint to subscribe on the price changes.
+  rpc StreamPrice(StreamPriceRequest) returns (stream StreamPriceResponse);
+
+  // Endpoint to subscribe on mutiple pairs price changes.
+  rpc StreamPrices(StreamPricesRequest) returns (stream StreamPricesResponse);
+
+  // Endpoint to query for price venue information.
+  rpc GetPriceVenueInfo(GetPriceVenueInfoRequest) returns (GetPriceVenueInfoResponse);
+
+  // Endpoint to query for available price pairs.
+  rpc GetPricePairs(GetPricePairsRequest) returns (GetPricePairsResponse);
+}
+
+// GetPriceRequest: Request fetching the pair price.
+message GetPriceRequest {
+  // pair: The specific market pair. e.g. ATOM/USDT
+  bolt.assets.v1.Pair pair = 1;
+}
+
+// GetPriceResponse: Response to GetPriceRequest.
+message GetPriceResponse {
+  // pair: The pair associated with the price. e.g. ATOM/USDT
+  bolt.assets.v1.Pair pair = 1;
+  // price: The price of the pair. e.g. 10.0
```

Comment:
```suggestion
  // price: The price of the pair, in rational values. e.g. 1250/13
```
To maintain best accuracy possible, let's use only rational values at the protobuf level for all such values. 
While `e.g 10` is valid, giving a more explicit example will be useful for new eyes looking into the proto files and trying to understand what is expected there.